prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -812,6 +812,28 @@ class ActionHistory : Logging {
                     }
                 }
                 jsonGen.writeEndArray()
+
+                jsonGen.writeArrayFieldStart("filteredReportRowObjects")
```

Comment:
Change this name to something a bit less code like.  E.g. remove Objects.  filteredReports?